ledger.sql

```diff
@@ -0,0 +1,12 @@
+CREATE  TABLE ledger
+(
+        PRIMARY KEY (ledger_id),
+
+        ledger_id CHAR(10) NOT NULL,
+			CONSTRAINT ledger_id_not_blank
+			CHECK (ledger_id <> '')
```

Comment:
Comma required after constraint, and not required between ledger_id and constraint.